# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/util-linux/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

Nur in util-linux-2.12h-patch/fdisk: cfdisk.c~.
Nur in util-linux-2.12h-patch/fdisk: fdisk.c~.
Nur in util-linux-2.12h-patch/fdisk: sfdisk.c~.
diff -dur util-linux-2.12h/mount/mount.c util-linux-2.12h-patch/mount/mount.c
--- util-linux-2.12h/mount/mount.c	2004-10-13 20:54:39.000000000 +0000
+++ util-linux-2.12h-patch/mount/mount.c	2004-10-23 03:16:30.310508504 +0000
@@ -1109,7 +1109,7 @@
  * Return 0 for success (either mounted sth or -a and NOAUTO was given)
  */
 static int
-mount_one (const char *spec, const char *node, const char *types,
+mount_one (const char *spec, char *node, const char *types,
 	   const char *opts, char *cmdlineopts, int freq, int pass) {
   int status, status2;
   const char *nspec;
Nur in util-linux-2.12h-patch/mount: mount.c~.
Nur in util-linux-2.12h-patch/mount: mount.c.rej.
